packages/core/src/address/address.advanced.ts

```diff
@@ -4,6 +4,23 @@ import { Client, KnownScript } from "../client";
 import { hexFrom } from "../hex";
 import { type AddressLike } from "./index";
 
+/**
+ * Parses an address string into an address information object.
+ *
+ * @param address - The address string to parse.
+ * @returns An object containing the address prefix, address format, and payload array.
+ *
+ * @throws Will throw an error if the address format is unknown.
+ *
+ * @example
+ * ```typescript
+ * const addressInfo = addressPayloadFromString("bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kygt080");
```

Comment:
This example doesn't seem to be correct.

Reply:
yep, seem like we should use ckb address here